dndsim/src/sim/events/AttackRollEvent.ts

```diff
@@ -11,6 +11,7 @@ export class AttackRollEvent {
     roll2: number = 0
     situationalBonus: number = 0
     minCrit?: number
+    private tags: Set<String> = new Set()
```

Comment:
It's possible that this should be on the attack itself, instead of on the attack roll, though I think you have plans to refactor that entire relationship anyway.